systemtest/ingest_test.go

```diff
@@ -300,3 +307,60 @@ func TestIngestPipelineBackwardCompatibility(t *testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+func TestMobileLogsWithGeoLocation(t *testing.T) {
+	systemtest.CleanupElasticsearch(t)
+	srv := apmservertest.NewUnstartedServerTB(t)
+	err := srv.Start()
```

Comment:
hmm why does this work without the capture_personal_data config?

Reply:
It seems like the capture_personal_data flag is enabled by default (at least for tests). I've added more info on it in [this other comment](https://github.com/elastic/apm-server/pull/11699#discussion_r1336089308).